src/Sylius/Bundle/CoreBundle/Fixture/Factory/OrderExampleFactory.php

```diff
@@ -0,0 +1,291 @@
+            ->setNormalizer('country', LazyOption::findOneBy($this->countryRepository, 'code'))
+        ;
+    }
+
+    protected function createOrder(ChannelInterface $channel, CustomerInterface $customer, CountryInterface $country): OrderInterface
+    {
+        $countryCode = $country->getCode();
+
+        $currencyCode = $channel->getBaseCurrency()->getCode();
+        $localeCode = $this->faker->randomElement($channel->getLocales()->toArray())->getCode();
+
+        /** @var OrderInterface $order */
+        $order = $this->orderFactory->createNew();
+        $order->setChannel($channel);
+        $order->setCustomer($customer);
+        $order->setCurrencyCode($currencyCode);
+        $order->setLocaleCode($localeCode);
+
+        $this->generateItems($order);
+
+        $this->address($order, $countryCode);
+        $this->selectShipping($order);
+        $this->selectPayment($order);
+        $this->completeCheckout($order);
+
+        return $order;
+    }
+
+    protected function generateItems(OrderInterface $order): void
+    {
+        $numberOfItems = random_int(1, 5);
+        $products = $this->productRepository->findAll();
+
+        for ($i = 0; $i < $numberOfItems; ++$i) {
+            /** @var OrderItemInterface $item */
+            $item = $this->orderItemFactory->createNew();
+
+            $product = $this->faker->randomElement($products);
+            $variant = $this->faker->randomElement($product->getVariants()->toArray());
+
+            $item->setVariant($variant);
+            $this->orderItemQuantityModifier->modify($item, random_int(1, 5));
+
+            $order->addItem($item);
+        }
+    }
+
+    protected function address(OrderInterface $order, string $countryCode): void
+    {
+        /** @var AddressInterface $address */
+        $address = $this->addressFactory->createNew();
+        $address->setFirstName($this->faker->firstName);
+        $address->setLastName($this->faker->lastName);
+        $address->setStreet($this->faker->streetAddress);
+        $address->setCountryCode($countryCode);
+        $address->setCity($this->faker->city);
+        $address->setPostcode($this->faker->postcode);
+
+        $order->setShippingAddress($address);
+        $order->setBillingAddress(clone $address);
+
+        $this->applyCheckoutStateTransition($order, OrderCheckoutTransitions::TRANSITION_ADDRESS);
+    }
+
+    protected function selectShipping(OrderInterface $order): void
+    {
+        if (!$this->orderShippingMethodSelectionRequirementChecker->isShippingMethodSelectionRequired($order)) {
+            $this->applyCheckoutStateTransition($order, OrderCheckoutTransitions::TRANSITION_SKIP_SHIPPING);
+
+            return;
+        }
+
+        /** @var ChannelInterface $channel */
+        $channel = $order->getChannel();
+        $shippingMethods = $this->shippingMethodRepository->findEnabledForChannel($channel);
+        if (count($shippingMethods) === 0) {
```

Comment:
Can we try use `Assert`(`Assert::notCount`)? Also, let's remove `Assert::notNull($shippingMethod, $this->generateInvalidSkipMessage('shipping', $channel->getCode()));`